src/adminCore/admin.controller.ts

```diff
@@ -53,7 +53,7 @@ export class DefaultAdminController {
 
   async getEntityWithRelations(adminEntity: AdminEntity, primaryKey: any) {
     const metadata = adminEntity.metadata
-    const relations = metadata.relations.map(r => r.propertyName)
+    const relations = metadata.relations.map((r) => r.propertyName)
```

Comment:
Can we deactivate this rule ? I prefer without the paranthesis

Reply:
Yes I set a new prettier config rule